src/routes/api/api-router.ts

```diff
@@ -82,16 +82,23 @@ ApiRouter.post(
 		const offset = Number(req.body.offset) || 0;
 		// only resync installations whose "updatedAt" date is older than x seconds
 		const inactiveForSeconds = Number(req.body.inactiveForSeconds) || undefined;
+		// How far back to fetch commit history(main and branch) in milliseconds.
+		const commitHistoryDepth = Number(req.body.commitHistoryDepth) || undefined;
```

Comment:
Went back and forth with commitHistoryDepth by itself or commitHistoryDepth + branchCommitHistoryDepth

Since there are two commit queries one for the main branch and one for other branches.

I ended up with one thats shared, because I was trying to limit the noise, but also the UI is only going to have 1 date picker(at this point 😂) so there was a consistency card to play there too....

Reply:
/nit

I am okay with it not too fuzzy, but personally find it `xxxDepth` is a number like indicate much layer was nested inside a directory.

If the `commitHistoryDepth` means "xxx in milliseconds", I find it easier to understand if named `commitHistoryDepthInMilliseconds`

Reply:
Fair, adding `milliseconds` os clear. naming is hard. lolol

The other option is align with the feature flag names `commitTimeLimit`

Reply:
change to be even even more explicit with commitsFromDate string